Content/Assets/CBR/Import/_CBX.as

```diff
@@ -0,0 +1,23 @@
+namespace Room { namespace rcbx { Template@ thisTemplate=Template();
```

Comment:
Put namespace openers, any code following them, and closing braces on separate lines, like so:
```namespace Room {
namespace RCBX {
Template@ thisTemplate=Template();
...
}
}```
You can choose to not indent namespaces.

Reply:
I will implement this into future iterations, with the typical closing comment at the end.